src/Uno.UI.Runtime.Skia.Gtk/Extensions/GtkDisplayInformationExtension.cs

```diff
@@ -18,7 +18,8 @@ internal class GtkDisplayInformationExtension : IDisplayInformationExtension
 	public GtkDisplayInformationExtension(object owner)
 	{
 		_displayInformation = (DisplayInformation)owner;
-		GtkHost.Current!.MainWindowShown += GtkDisplayInformationExtension_MainWindowShown;
+		//TODO:MZ:
+		//GtkHost.Current!.MainWindowShown += GtkDisplayInformationExtension_MainWindowShown;
```

Comment:
Display information extension will depend on initial window, but not be shared (multiple instances instead)